src/main/java/org/broadinstitute/hellbender/tools/spark/sv/BreakpointAllele.java

```diff
@@ -0,0 +1,41 @@
+package org.broadinstitute.hellbender.tools.spark.sv;
+
+import org.broadinstitute.hellbender.utils.SimpleInterval;
+
+import java.util.Objects;
+
+class BreakpointAllele {
+    SimpleInterval leftAlignedLeftBreakpoint;
```

Comment:
I think that that `Allele` class has a lot of stuff that's not relevant to this class, and vice versa, so it might be more trouble than its worth to make one extend the other. For example, the htjdk class does not have any concept of genomic location, and is just concerned with the bases content of a ref or alternate allele. Perhaps this class shouldn't be called `BreakpointAllele`. I'll see if I can come up with different names that would make sense.